### PR TITLE
feat(chart): `ingressClassName` and `tls` for ingress

### PR DESCRIPTION
**Pull request by jjangga0214**

### CODE REVIEW

File: operations/pyroscope/helm/pyroscope/templates/ingress.yaml [hunk 1]
```diff
@@ -11,6 +11,13 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
+  ingressClassName: {{ .Values.ingress.className }}
+  {{- end }}
+  {{- with .Values.ingress.tls }}
+  tls:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```
Comment (kolesnikovae) [inline operations/pyroscope/helm/pyroscope/templates/ingress.yaml:20]: I'm not very sure we want the capability version check as we unconditionally set the API version to `networking.k8s.io/v1` (1.19)

**Review by kolesnikovae — COMMENTED** [review]
Thank you for the contribution @jjangga0214! 🎉 

Could you please add empty attributes to the value file and run `helm/docs`?

**Comment by kolesnikovae** [comment]
Hello @jjangga0214, could you please resolve the review comments? I think that the change might be also interesting for others, therefore I'd like to get it merged

**Comment by jjangga0214** [comment]
@kolesnikovae Done :) 
Thank you for the review and maintenance!

**Comment by kolesnikovae** [comment]
Thank you @jjangga0214! Could you also please run `helm/check` and push the updated templates?

**Review by simonswine — APPROVED** [review]
It was useful to have that sooner, so I ran `helm check` and commited the result.

lgtm

Thanks for the contriubtion @jjangga0214  🙂

**Pull request merged**